frontend/test/__support__/e2e/helpers/e2e-email-helpers.js

```diff
@@ -53,19 +53,21 @@ export const clearInbox = () => {
 };
 
 export const openEmailPage = emailSubject => {
-  cy.window().then(win => (win.location.href = "http://localhost"));
```

Comment:
The problem was that this string was using `http://localhost` instead of explicit `http://localhost:80`.
That's why I didn't see it in #28310.

Reply:
Yeah, this is easily missable.